examples/c/spi.c

```diff
@@ -22,11 +22,10 @@
 #define SPI_BUS 0
 
 /* SPI frequency in Hz */
-#define SPI_FREQ 400000
+#define SPI_FREQ 100000
 
-uint16_t pat[] = { 0xaa01, 0x5502, 0xaa03, 0x5504, 0xaa05, 0x5506, 0xaa07, 0x5508 };
```

Comment:
I prefer to reserve this pattern and it's useful for us to know how to control the LED matrix.

Reply:
Yes, I agree. It's useful.